route_info.go

```diff
@@ -103,7 +103,6 @@ func (ri RouteInfo) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 
 	events.EmitPayload(EvtRouteStarted, payload)
 	err := a.Middleware.handler(ri)(c)
-	c.Flash().persist(c.Session())
```

Comment:
It was not clear why this is needed, so I removed it. `persist()` no longer saves Session, so is this needed? Only reference is: https://github.com/gobuffalo/buffalo/pull/2054